### PR TITLE
Create apa6-for-register-studies.csl

### PR DESCRIPTION
**Pull request by elenlefoll**
First version of a citation stylesheet for Register Studies (John Benjamins)

### CODE REVIEW

**Comment by rmzelle** [comment]
@elenlefoll, thanks. I see that https://benjamins.com/catalog/rs says "The full reference list should follow guidelines provided by the American Psychological Association (6th edition). A few examples follow; please consult the APA manual for full details."

But it looks like you modified the APA 6th edition style format. Are you sure the journal uses its own custom format, and doesn't accept plain APA 6th edition style references?

**Comment by elenlefoll** [comment]
Hi @rmzelle, yes, that's right, the journal follows APA6 for the bibliography **but** the in-text citations are in a different style. All this modified stylesheet does different to the APA6 template I used is the formatting of the in-text citations.

File: register-studies.csl [hunk 1]
```diff
@@ -0,0 +1,1512 @@
...
+    <group>
+      <choose>
+        <if locator="page" match="none">
+          <label variable="locator" form="short" suffix=" "/>
+        </if>
+      </choose>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="2" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
```
Comment (adam3smith) [inline register-studies.csl:1469]: The `et-al-use-first="2"` here seems wrong to me. The RS styleguide has
> (Johnson et al. 2014: 43)

Your version produces

>(Johnson, Meyer, et al. 2014: 43)

**Comment by adam3smith** [comment]
@elenlefoll -- I have one remaining question above, everything else looks good to me. Thanks!

File: register-studies.csl (hunk 1 above)
Comment (elenlefoll) [reply register-studies.csl:1469]: Thanks for picking this up. You're absolutely right, it should be: et-al-use-first="1".

Comment (rmzelle) [reply register-studies.csl]: I removed the comma in front of "et al." as well.

**Comment by rmzelle** [comment]
@elenlefoll, thanks!

File: register-studies.csl (hunk 1 above)
Comment (elenlefoll) [reply register-studies.csl]: Thank you!

**Pull request merged**